README.md

```diff
@@ -235,7 +235,10 @@ There are three ways to make cookies available to `leetgo`:
 
 - Read cookies from browser automatically.
   
-  Currently, Chrome and FireFox are supported, if you want to support other browsers, please let us know!
+  Currently, `leetgo` supports Chrome, FireFox, and Safari*.
+  If you want to support other browsers, please let us know!
+
+  *For Safari on MacOS, you may need to grant `Full Disk Access` privilege to your terminal app which you would like to run `leetgo`.
```

Comment:
How about using [footnote](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/) for this style of comments?

Reply:
You are right. It would be better.

Reply:
and `RADME_zh.md` should be updated too 😆

Reply:
Well, it seems that GitHub doesn't support footnote.

Reply:
Maybe we can change the content related to `browsers` to a separate block. Then all the browsers could be listed as a bullet list.

Reply:
> Well, it seems that GitHub doesn't support footnote.

I fixed it by adding a colon, now it is rendered nicely:
![image](https://user-images.githubusercontent.com/10510431/216537374-422e40b4-668f-4874-beaa-f2835dc7636b.png)

> Maybe we can change the content related to browsers to a separate block. Then all the browsers could be listed as a bullet list.

That's okay to me too.

Reply:
okay, I will add zh version later.